Makefile

```diff
@@ -286,3 +286,11 @@ mpi-operator-crd:
 	GOPATH=/tmp GO111MODULE=on $(GO_CMD) install github.com/kubeflow/mpi-operator/cmd/mpi-operator@$(MPI_VERSION)
```

Comment:
can you update this to use `go list` as well? In a separate PR

Reply:
will do #801